src/constants.rs

```diff
@@ -49,4 +49,4 @@ pub const ID_CERTIFICATE_VALIDITY_YEARS: i32 = 15;
 
 pub const BGP_RIS_REFRESH_MINUTES: i64 = 60;
 
-pub const HTTTP_CLIENT_TIMEOUT_SECS: u64 = 120;
+pub const HTTTP_CLIENT_TIMEOUT_SECS: u64 = 900;
```

Comment:
This still seems to be 120 on HEAD of `main`, i.e. the issue doesn't just exist in 0.7.4 but also in newer versions. Is it the intention to make this config file configurable as part of #740?